Account/FxAClient10.swift

```diff
@@ -48,6 +60,24 @@ public struct FxASignResponse {
     }
 }
 
+public struct FxAStatusResponse {
+    let exists: Bool
+    let locked: Bool
+
+    init(exists: Bool, locked: Bool) {
+        self.exists = exists
+        self.locked = locked
```

Comment:
Thanks! While here, I also fixed up a number of unnecessary `struct` `init`s we had in this file.